polars/tests/it/lazy/aggregation.rs

```diff
@@ -95,3 +95,55 @@ fn test_apply_multiple_error() {
         .collect()
         .unwrap();
 }
+
+#[test]
```

Comment:
I am not really sure about this test. I think just changing the assert is fine

Reply:
Thanks @ritchie46 , I've removed the integration unittest. Although personally I think it could be a good one as it touches on multiple important parts. If you change your mind lmk I will plug it back in.